README.md

```diff
@@ -24,70 +34,39 @@ SmartBugs requires [Python3](https://www.python.org). To install all the require
+              --tool TOOLS          # the list of tools to use for the analysis (all to use all of them) 
+              --skip-existing       # skip the execution that already has results
+              --processes PROCESSES # the number of process to use during the analysis (by default 1)
+```
 
 For example, we can analyse all contracts labelled with type `reentrancy` with the tool oyente by executing:
 
+```bash
+python3 smartBugs.py --tool oyente --file dataset/reentrancy
 ```
-python3 smartBugs.py --tool oyente --dataset reentrancy
-```
-
-By default, results will be placed in the directory `results`. 
-
-
-### Adding your tool or any third-party tool to SmartBugs
-
-You will need to add a configuration file in `config/tools` and define the docker image to use and command to run. The config file should follow a structure similar to the following:
-  ```
-  docker_image:
-    default: primary docker image [REQUIRED]
-    solc<5: [OPTIONAL]
-  cmd: command to run analysis [REQUIRED]
-  info: info about the tool [OPTIONAL]
 
-  output_in_files:
-    folder: if the tool does not log results in console, you should provide the path file inside the docker image to get the results [OPTIONAL]
-  ```
-Please check the provided config files for more concrete examples.
+By default, results will be placed in the directory results. 
+The results of the analysis on 47,587 contracts is available here: https://github.com/smartbugs/smartbugs-wild
 
+## Known Limitations
 
-### Adding new named sets 
-
-To add a new named set, edit the configuration file `dataset.yml` (in the folder `config/dataset`) and add the named path you want (can be files or dirs):
-
-The config file should follow a structure similar to:
-  ```
-  type_1: PATH
-  type_2:
-        - PATH
-        - PATH
-  ```
-Please check the provided config files for more concrete examples.
+When running a tool the user must be aware of the solc compatibility. Due to the major changes introduced in solidity v0.5.0, we provide the option to pass another docker image to run contracts with solidity version below v0.5.0. However, please note that there may still be problems with the solidity compiler when compiling older versions of solidity code. 
 
-## Known limitations
+## Smart Contracts dataset
 
-When running a tool the user must be aware of the solc compatibility. Due to the major changes introduced in solidity v0.5.0, we provide the option to pass another docker image to run contracts with solidity version below v0.5.0. However, please note that there may still be problems with the solidity compiler when compiling older versions of solidity code. 
+We propose two dataset a smart contract dataset with SmartBugs.
+The first dataset contains 69 annotated contracts to evaluate the accuracy of analysis tools.
+The second dataset contains 47,518 unique contract from the Ethereum network.
 
-## Vulnerabilities
+### SB Curated
```

Comment:
should we say what SB statnds for? Or is it clear?

Reply:
I think it should be ok because, it is just the name of the benchmark. I propose to merge and we can still improve it later.